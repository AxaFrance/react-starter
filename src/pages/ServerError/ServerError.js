import React from 'react';
import './ServerError.scss';

const ServerError = () => (
  <>
    <div className="container af-container--notfound">
      <h1 className="af-notfound__title">
        <div className="af-notfound__title-covernumber">
          <span className="af-notfound__title-number">Err</span>
        </div>
      </h1>
      <p className="af-notfound__message">
        The page you are looking for has a problem!
      </p>
    </div>
  </>
);

export default ServerError;
