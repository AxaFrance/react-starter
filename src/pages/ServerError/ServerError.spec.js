import React from 'react';
import { render } from '@testing-library/react';
import ServerError from './ServerError';

it('1. Renders ServerError component without crashing', () => {
  const { asFragment } = render(<ServerError title="test" />);
  expect(asFragment()).toMatchSnapshot();
});
