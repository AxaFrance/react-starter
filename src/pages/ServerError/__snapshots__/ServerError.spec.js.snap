// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1. Renders ServerError component without crashing 1`] = `
<DocumentFragment>
  <div
    class="container af-container--notfound"
  >
    <h1
      class="af-notfound__title"
    >
      <div
        class="af-notfound__title-covernumber"
      >
        <span
          class="af-notfound__title-number"
        >
          Err
        </span>
      </div>
    </h1>
    <p
      class="af-notfound__message"
    >
      The page you are looking for has a problem!
    </p>
  </div>
</DocumentFragment>
`;
